packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/editInfoUtils/getLastZIndex.ts

```diff
@@ -10,9 +10,9 @@ export default function getLatestZIndex(editorDiv: HTMLElement) {
     let child: HTMLElement | null = editorDiv;
     let zIndex = 0;
     while (child && getTagOfNode(child) !== 'BODY') {
-        const childZIndex = child.style.zIndex || getComputedStyle(child).zIndex;
-        if (childZIndex) {
-            zIndex = parseInt(child.style.zIndex);
+        const childZIndex = parseInt(child.style.zIndex || getComputedStyle(child).zIndex, 10);
+        if (childZIndex && !isNaN(childZIndex)) {
```

Comment:
If childZIndex is NaN, the condition is already false, so no need `&& !isNaN(childZIndex)`